server/src/main/java/org/elasticsearch/index/store/Store.java

```diff
@@ -451,6 +451,8 @@ public static MetadataSnapshot readMetadataSnapshot(
             return MetadataSnapshot.loadFromIndexCommit(null, dir, logger);
         } catch (IndexNotFoundException ex) {
             // that's fine - happens all the time no need to log
+        } catch (CorruptIndexException e) {
+            logger.info(() -> shardId + "is corrupted", e);
```

Comment:
Suggest harmonising with the `ShardLockObtainFailedException` case below (also `"is corrupted"` needs a leading space):

```suggestion
        } catch (CorruptIndexException ex) {
            logger.info(() -> format("%s: corrupted", shardId), ex);
```